.github/workflows/build_and_test.yml

```diff
@@ -77,8 +77,51 @@ jobs:
       run: echo "::error file=.github/workflows/build_and_test.yml::File not included in any filter" && false
       if: ${{ !contains(steps.filter.outputs.*, 'true') }}
 
-  rust:
-    name: Rust
+  rust-stable:
+    name: Rust stable
+
+    runs-on: ubuntu-latest
+
+    needs: changes
+
+    if: ${{ needs.changes.outputs.rust == 'true' }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: sudo apt-get install gcc-multilib
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        components: rustfmt, clippy
+        target: i686-unknown-linux-gnu
+
+    - name: Check for duplicate dependencies
+      run: |
+        DUPS="$(cargo tree -d -e normal --workspace)"
+        echo "$DUPS"
+        test -z "$DUPS"
```

Comment:
I don't think we need to do this again; anyone who's using this as a Rust library will either want to redo the check anyway or won't care.